content/learn/07.built-in-libraries/02.i2s/i2s.md

```diff
@@ -150,26 +165,67 @@ I2S.availableForWrite()
 none
 
 #### Returns
-the number of bytes available to write.
+size_t
```

Comment:
why?

Reply:
I'm not sure I get it. Why wouldn't you want to specify the return type for a function in the documentation? Some other pages in the Arduino docs lack this information as well, and I have sometimes been forced to read the source.